caterva2/api.py

```diff
@@ -1020,10 +1064,8 @@ def download(self, localpath=None):
         >>> file = root['ds-1d.b2nd']
         >>> file.download()
         PosixPath('example/ds-1d.b2nd')
-        >>> file.download('mypath')  # assuming 'mypath' exists
-        PosixPath('mypath/ds-1d.b2nd')
-        >>> file.download('mypath/myds.b2nd')  # 'mypath' will be created if it doesn't exist
-        PosixPath('mypath/myds.b2nd')
+        >>> file.download('mypath')
+        PosixPath('mypath')
```

Comment:
Perhaps a small explanation (a comment would be fine) on that mypath is meant to be a dir.  Better yet, why not using `mydir` instead?

Reply:
Fixed in 0fe79ea0f27976e7d428308106ade05bc649e08b